collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+
+### Most installs are from Brazil \+ India
+
+Brazil and India make up **26%** of installs with **17%** coming from Brazil and **9%** coming from India. 
+
+<img alt="Android editor device installs percentage by country" src="/storage/blog/godotcon-2025-android-xr/godot-android-editor-countries-installs-2025.webp" />
+
+From this data point, we can observe how the Android editor is **helping to lower the barrier of entry to game dev** by allowing users in Brazil, India, and other countries to make use of **cheaper** and more **accessible** Android devices for game development.
+
+### Most installs are on phones\!
+
+According to the data, **\~87%** of installs **are on phones\!**   
+
+<img alt="Android editor device installs by form-factor percentage" src="/storage/blog/godotcon-2025-android-xr/godot-android-editor-device-type-percentage-2025.webp" />
+
+In the [release blog post](https://godotengine.org/article/android_godot_editor_play_store_beta_release/), we called out the Android editor as being optimized for large form-factor devices (e.g.: foldables, tablets, Chromebooks), so we expected those types of devices to be driving the installs. However, from the data it can be seen that tablets and Chromebooks only make up **\~10.6%** of installs. So while the experience is still unoptimized for phones, for a vast number of users, the Android editor has a valuable use and addresses a valid need on regular Android phones.
+
+These data points are driving some of our upcoming work, as in future releases we’ll strive to close the feature parity gap between the Android and desktop versions of the editor, as well as work to improve the Android editor development experience on small form-factor devices like phones.
+
+## XR editor
+
+ <video autoplay loop muted playsinline title="Using Meta Quest seamless multitasking to modify XR game in real-time">
+  <source src="/storage/blog/godot-editor-horizon-store-early-access-release/xr_game_live_modifications.mp4?1" type="video/mp4">
+</video>
+
+[Released in 2024](https://godotengine.org/article/godot-editor-horizon-store-early-access-release/) to [great reception](https://www.uploadvr.com/godot-engine-standalone-on-quest-horizon-os/), the XR editor is available on the [Meta Horizon store](https://www.meta.com/experiences/godot-game-engine/7713660705416473/) and on [Godot’s download page](https://godotengine.org/download/android/) with support for Quest 3, Quest 3S, and Quest Pro devices.  
+   
+To date, the Godot XR editor is the **only** **full game engine** **running entirely on standalone XR devices** and providing users with the ability to **develop apps and games with nothing but a standalone XR device**\!   
+To highlight that fact, David & Logan recently completed [a game jam](https://dsnopek.itch.io/expedition-to-blobotopia) [using nothing but the XR editor](https://dsnopek.itch.io/expedition-to-blobotopia/devlog/949090/making-a-godot-game-entirely-in-vr-gwj-81-devlog-nr-1)\!
```

Comment:
```suggestion
To date, the Godot XR editor is the **only** full game engine **running entirely on standalone XR devices** and providing users with the ability to **develop apps and games with nothing but a standalone XR device**. To highlight that fact, David & Logan recently completed [a game jam](https://dsnopek.itch.io/expedition-to-blobotopia) [using nothing but the XR editor](https://dsnopek.itch.io/expedition-to-blobotopia/devlog/949090/making-a-godot-game-entirely-in-vr-gwj-81-devlog-nr-1)\!
```

These paragraphs can be merged into one and remove the ! at the end of one of them